Makefile

```diff
@@ -24,7 +24,7 @@ else
 test-gt = $(shell test $(strip $1)0 -gt $(strip $2)0 && echo y)
 endif
 
-TFW_CFLAGS = $(DEFINES) -Werror
+TFW_CFLAGS = $(DEFINES) -Werror -Wno-missing-prototypes -Wno-error=missing-declarations -Wno-missing-declarations
```

Comment:
In general needs because of `DO_INIT` macro, if we rewrite this macro we will not need this.

Reply:
Removed, code has been rewritten